src/run.py

```diff
@@ -2,4 +2,4 @@
 from os import environ
 
 if __name__ == "__main__":
-    app.run(host="0.0.0.0", port=environ["PORT"])
+    app.run(host="0.0.0.0", port=environ["PORT"], debug=True)
```

Comment:
leave this out

Reply:
this is for hot reloading

Reply:
yeah but we don't want this in production